12_glob_lab/globlab.txt

```diff
@@ -4,26 +4,31 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 1. Type in a glob pattern to match:
 
   1. All files whose name starts with a lowercase letter
-  Answer: 
+  Answer: [a-z]*
 
+  Ofir>> for some reason the shell is not case sensitive so also upper case letters are matched, same for the next quesiton. 
+  
   2. All files whose name starts with an uppercase letter
-  Answer:
+  Answer: [A-Z]*
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
-  Answer:
+  Answer: *[aeiou].*
```

Comment:
a file does not have to contain a dot in their name, for example: ```/etc/passwd``` is a simple text file